packages/rocketchat-lib/server/methods/setUsername.coffee

```diff
@@ -11,7 +11,7 @@ Meteor.methods
 			return username
 
 		if not /^[0-9a-zA-Z-_.]+$/.test username
-			throw new Meteor.Error 'username-invalid'
+			throw new Meteor.Error 'username-invalid', TAPi18next.t('project:Username_invalid', username)
```

Comment:
Still new to Meteor, unsure which pieces fit where. I assumed the client data got passed down and translated then passed back up to the server.

Sorry!  I'll read the docs better next time.